src/styles.ts

```diff
@@ -0,0 +1,101 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
can we name all our files .tsx? This avoids situations where we rename a file .ts -> .tsx after we realize we need to put jsx in it.

also, the linters that are set up are currently only checking .tsx files. alternatively we can just add .ts  patterns.

Reply:
Sure thing!